examples/resources/meshstack_payment_method/resource.tf

```diff
@@ -0,0 +1,22 @@
+data "meshstack_workspace" "example" {
+  metadata = {
+    name = "my-workspace-identifier"
+  }
+}
+
+resource "meshstack_payment_method" "example" {
+  metadata = {
+    name               = "my-payment-method"
+    owned_by_workspace = data.meshstack_workspace.example.metadata.name
+  }
+
+  spec = {
+    display_name    = "My Payment Method"
+    expiration_date = "2025-12-31T23:59:59Z"
```

Comment:
f: I tried it locally with this expiration date, and it didn't work. looks like you need specify the date only:
```suggestion
    expiration_date = "2025-12-31"
```

<img width="1221" height="203" alt="Image" src="https://github.com/user-attachments/assets/57e604e3-ceee-42a1-a926-3fe87d263714" />